ibis/backends/tests/test_vectorized_udf.py

```diff
@@ -516,30 +513,27 @@ def test_elementwise_udf_overwrite_destruct_and_assign(backend, alltypes):
     backend.assert_frame_equal(result, expected, check_like=True)
 
 
-@pytest.mark.only_on_backends(['pandas', 'pyspark', 'dask'])
-@pytest.mark.xfail_unsupported
-@pytest.mark.min_spark_version('3.1')
-def test_elementwise_udf_destruct_exact_once(backend, alltypes):
-    with tempfile.TemporaryDirectory() as tempdir:
-
-        @elementwise(
-            input_type=[dt.double],
-            output_type=dt.Struct(['col1', 'col2'], [dt.double, dt.double]),
-        )
-        def add_one_struct_exact_once(v):
-            key = v.iloc[0]
-            path = Path(f"{tempdir}/{key}")
-            assert not path.exists()
-            path.touch()
-            return v + 1, v + 2
-
-        result = alltypes.mutate(
-            add_one_struct_exact_once(alltypes['index']).destructure()
-        )
+# @pytest.mark.only_on_backends(['pandas', 'pyspark', 'dask'])
+# @pytest.mark.xfail_unsupported
+# @pytest.mark.min_spark_version('3.1')
+# def test_elementwise_udf_destruct_exact_once(backend, alltypes, tmp_path):
```

Comment:
Since this PR changes the IR the execution model changes as well, which means that currently we cannot reuse the results coming from the same UDF call for separate operations.

I think this should be rather implemented in the execution model of pandas since we should be able to reuse results from different branches of the expression tree.

Reply:
It is.

Reply:
@gerrymanoim @jreback @emilyreff7 @icexelloss Wondering if y'all are affected by this.